src/material-examples/expansion-steps/expansion-steps-example.html

```diff
@@ -0,0 +1,70 @@
+<md-accordion class="example-headers-align">
+  <md-expansion-panel [expanded]="step === 0" (opened)="openEvent(0)" hideToggle="true">
```

Comment:
Your html point is good... maybe wait for a team member before dropping? But yeah, I think the styles can be cleaned up a little anyway 😁